src/iceberg_manifest.cpp

```diff
@@ -58,6 +60,7 @@ idx_t IcebergManifestV2::ProduceEntries(DataChunk &chunk, idx_t offset, idx_t co
 		manifest.manifest_path = manifest_path[index].GetString();
 		manifest.content = IcebergManifestContentType(content[index]);
 		manifest.sequence_number = sequence_number[index];
+		manifest.added_rows_count = added_rows_count[index];
```

Comment:
Judging from https://iceberg.apache.org/spec/#manifest-lists, this looks like we should also use the `existing_rows_count`, for files that aren't added by the snapshot, but are still part of the manifest (?)